### DISCUSSION TITLE
pipelines not ingesting following netflow setup, solution found.

### DISCUSSION THREAD

**Discussion by rock-kobs** [Security-Onion-Solutions/securityonion#9900] [General] [answered]
following the setup instruction for netflow when you do copy and paste of the instruction the step where you have to ingest the pipelines modules does not work as it used to, there's a solutions on Weslambert github page however if you looking for quick command from the page that ingest the netflow command type this;
working command : sudo docker exec -i so-filebeat filebeat setup modules --pipelines --modules netflow -M "netflow.log.enabled=true" -c /usr/share/filebeat/module-setup.yml

wrong command: sudo docker exec -i so-filebeat filebeat setup modules -pipelines -modules netflow -c /usr/share/filebeat/module-setup.yml

note## the difference in the commands

hope it helps,

**Comment by andre-dessert** [comment, marked as answer]
Solution is in the post: 
```
...if you looking for quick command from the page that ingest the netflow command type this;
working command : sudo docker exec -i so-filebeat filebeat setup modules --pipelines --modules netflow -M "netflow.log.enabled=true" -c /usr/share/filebeat/module-setup.yml
```

We will be updating the docs to reflect this solution.
